airflow/www_rbac/package.json

```diff
@@ -57,11 +56,11 @@
     "bootstrap-toggle": "^2.2.2",
     "d3": "^3.4.4",
     "d3-tip": "^0.9.1",
-    "dagre-d3": "^0.4.18",
+    "dagre-d3": "^0.6.1",
```

Comment:
Unfortunately this breaks the www_rbac graph view, see JS error below. I cleaned `airflow/www_rbac/node_modules/` and run `airflow/www_rbac/compile_assets.sh`.

We still use d3 version 3, so dagre-d3 version 0.4.18 is the last that can be used, see also https://github.com/dagrejs/dagre-d3/issues/309

If I revert back to dagre-d3 0.4.18 the graph view works. Should I create a PR to revert back? But I only have limited npm knowledge, not sure how to prevent the audit warnings. Or is there a plan to update d3?

```
dagre-d3.min.js:24 Uncaught TypeError: svgEdgeLabels.exit is not a function
    at createEdgeLabels (dagre-d3.min.js:24)
    at Array.fn (dagre-d3.min.js:90)
    at Array.Co.call (d3.min.js:3)
    at graph?dag_id=demo:1131
```

Reply:
Hm, there is a fix https://github.com/dagrejs/dagre-d3/pull/313 which is included in dagre-d3 0.6.3. However then I get a different error: https://github.com/dagrejs/dagre-d3/issues/309#issuecomment-450680411